torch/_decomp/decompositions.py

```diff
@@ -52,31 +52,46 @@ def decrease_prec(x):
 
     return inner
 
+# This expands x until x.dim() == dim. Might be useful as an operator
+def _unsqueeze_to_dim(x: Tensor, dim: int):
+    for _ in range(dim - x.dim()):
+        x = x.unsqueeze(-1)
+    return x
+
 
 @register_decomposition(aten.tanh_backward)
 @cast_for_opmath
-def tanh_backward_decomposition(out_grad: Tensor, y: Tensor):
-    return out_grad * (1 - y * y)
+def tanh_backward(out_grad: Tensor, y: Tensor):
+    return out_grad * (1 - y * y).conj_physical()
```

Comment:
I wanted to say "why can't we use conj" but the problem is that at the point decompositions run there is no way to apply conjugate views anymore? We should at least make sure calling conj() errors (but it would nice to make this work). cc @bdhirsh since functionalization kind of has a similar problem.